apps/explorer/src/types/assets.ts

```diff
@@ -16,39 +16,63 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { TokenInfo } from '@alephium/token-list'
-import { FungibleTokenMetaData, NFTMetaData } from '@alephium/web3'
+import { NFTTokenUriMetaData } from '@alephium/shared'
+import { FungibleTokenMetadata, NFTMetadata as NFTMetadataBase } from '@alephium/web3/dist/src/api/api-explorer'
 
 import client from '@/api/client'
 
 export type AssetType = Awaited<ReturnType<typeof client.node.guessStdTokenType>>
 
 export type AssetBase = { id: string; type: AssetType }
 
-export type FungibleTokenMetadata = Omit<FungibleTokenMetaData, 'totalSupply'> & { id: string; verified: boolean }
+export type FungibleTokenMetadataBase = Omit<FungibleTokenMetadata, 'totalSupply' | 'decimals'> & {
+  verified: boolean
+  decimals: number
+  logoURI?: string
+}
+
+export type VerifiedFungibleTokenMetadata = TokenListTokenInfo & {
+  type: 'fungible'
+  verified: true
+}
+
+export type UnverifiedFungibleTokenMetadata = FungibleTokenMetadataBase & { type: 'fungible'; verified: false }
 
-export type VerifiedFungibleTokenMetadata = TokenInfo & { type: 'fungible'; verified: true }
+export type NFTMetadata = NFTMetadataBase & { id: string; type: 'non-fungible'; file?: NFTFile; verified: boolean }
 
-export type UnverifiedFungibleTokenMetadata = TokenInfo & { type: 'fungible'; verified: false }
+export type NFTMetadataWithFile = Omit<NFTMetadata, 'file'> & { file: NFTFile }
 
-export type UnverifiedNFTMetadata = NFTMetaData & { id: string; type: 'non-fungible'; verified: false }
+export type UnverifiedNFTMetadata = Omit<NFTMetadata, 'verified'> & { file?: NFTFile; verified: false }
```

Comment:
I wonder if it's necessary to first omit the `verified` prop. Wouldn't it just get overridden with this?

```ts
export type UnverifiedNFTMetadata = NFTMetadata & { file?: NFTFile; verified: false }
```